lua/xcodebuild/ui/pickers.lua

```diff
@@ -258,6 +259,13 @@ function M.select_scheme(schemes, callback, opts)
   M.show("Select Scheme", schemes or {}, function(value, _)
     projectConfig.settings.scheme = value
     projectConfig.save_settings()
+
+    if config.update_build_server_on_scheme_change then
+      vim.fn.jobstart(
+        "xcode-build-server config " .. projectConfig.settings.projectCommand .. " -scheme " .. value
+      )
```

Comment:
Could you please move it to `integrations/xcode_build_server.lua`. I know that it's just one function, but I'd prefer to keep external dependencies separately.

Please make sure to add tags describing this module for help usage (just see any other file for the reference).

Reply:
Please also wrap here the scheme with `'` as it could contain white spaces:

```lua
" -scheme '" .. value .. "'"
```